src/sankoff_handler.hpp

```diff
@@ -87,19 +95,28 @@ class SankoffHandler {
     return partials_at_pattern;
   }
 
-  // sum p-partials for right and left children of node 'node_id'
+  // Fill in leaf-values for P partials.
+  void GenerateLeafPartials();
+
+  // Sum p-partials for right and left children of node 'node_id'
   // In this case, we get the full p-partial of the given node after all p-partials
   // have been concatenated into one SankoffPartialVector
   EigenVectorXd TotalPPartial(NodeId node_id, size_t site_idx);
 
-  // fill in leaf-values for P partials
-  void GenerateLeafPartials();
-
-  // calculate the partial for a given parent-child pair
+  // Calculate the partial for a given parent-child pair
   EigenVectorXd ParentPartial(EigenVectorXd child_partials);
 
+  // Populate rootward parsimony PV for node.
```

Comment:
This is an example of something that's going to get changed when we are moving partial vectors to edges, right? In that case I think this will make more sense.

Reply:
Yes it will.